model_compression_toolkit/keras/quantization_facade.py

```diff
@@ -29,9 +29,12 @@
 
 import importlib
 
+from model_compression_toolkit.common.hardware_representation.hardware2framework import FrameworkHardwareModel
+
 if importlib.util.find_spec("tensorflow") is not None\
         and importlib.util.find_spec("tensorflow_model_optimization") is not None:
     import tensorflow as tf
+    from model_compression_toolkit.hardware_models.keras_hardware_model.keras_default import KERAS_DEFAULT_MODEL
```

Comment:
Please define this  KERAS_DEFAULT_MODEL where using get_model function

Reply:
fixed